server/botany/settings.py

```diff
@@ -11,7 +11,7 @@
 """
 
 import os
-from datetime import datetime, timedelta, timezone
+from datetime import datetime, timedelta, timezone, date
```

Comment:
date is never used in the changeset. Make sure to run `fixup.sh` in the root, perhaps as a pre-commit hook. That'll run flake8 which would detect such issues early.